cmd/cmd.go

```diff
@@ -69,7 +70,7 @@ func (cmd *Cmd) Main(args []string) (err error) {
 	g := &ir.Comp.Globals
 
 	var set, clear Options
-	var repl, forcerepl = true, false
+	repl, forcerepl := true, false
```

Comment:
Sorry, this was my editor auto-fixing things.  I can revert if you have strong feelings.